### PR TITLE
Media query for desktop added

### PR DESCRIPTION
**Pull request by baezluise**
Style adapted to desktop version

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -437,3 +423,145 @@ form {
...
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  gap: 30px;
+}
+
+@media only screen and (min-width: 768px) {
+  #welcome {
```
Comment (joskalenda) [inline styles.css:447]: - [ ] Great design @baezluise, you have done awesome progress but there are a few things in the design that need to be fixed, see the below comments

FIgma design


 
- Comparing this design to yours, I think you need to fix some points like the space between the title and the arrow icon, the arrow icon itself, and the text below, please refer to the image to update yours below.

YOUR DESIGN




- The second one is this here below, see how the space between the button and the footer is so small, consider checking the design one more to update that part, please



The figma

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @baezluise , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
| IMAGE |

## Highlights

- [ ]... ✅ Great design
- [ ].. ✅  Clean code
- [ ].... ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Qoosim) [reply styles.css:447]: - [ ] These required changes have not been implemented. Kindly attend to it

> * [ ]  Great design @baezluise, you have done awesome progress but there are a few things in the design that need to be fixed, see the below comments
> 
> FIgma design
> 
> 
> 
> * Comparing this design to yours, I think you need to fix some points like the space between the title and the arrow icon, the arrow icon itself, and the text below, please refer to the image to update yours below.
> 
> YOUR DESIGN
> 
> 
> 
> * The second one is this here below, see how the space between the button and the footer is so small, consider checking the design one more to update that part, please
> 
> 
> 
> The figma
> 
>

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @baezluise, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GiHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -147,9 +155,13 @@ <h2 class="card-name-2">Skills</h2>
     </div>
     </ul>
     </section>
+<div class="contact-container-2">
+  <div class="contact-container">
+    <div class="para-form-container">
+    <div class="para-form">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
```
Comment (huseyinbiyik) [inline index.html:161]: While you are fixing the requested changes, unfortunately, you broke the previous design. "I'm always interested in hearing about new projects, so if you'd like to chat please get in touch." section is not aligned to the center horizontally anymore. Kindly fix that.
It was like that before:
Screenshot 2023-04-06 at 12 11 51 AM
After fixing the requested changes from the reviewers, now:
Screenshot 2023-04-06 at 12 12 08 AM
How it should be:
Screenshot 2023-04-06 at 12 12 37 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @baezluise 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @baezluise, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**